### PR TITLE
fix(dashboard): expose data flow graph summary state

### PR DESCRIPTION
**Pull request by jeong-sik**
## Summary
- export DataFlowGraph summary helpers and metadata for node/edge counts, status, totals, and missing edges
- render a compact summary strip plus empty-state metadata for graph QA and dashboard inspection
- expose node/edge data attributes and fix the default edge stroke to use the semantic accent token

## Why it matters
DataFlowGraph could look empty in browser QA because the default edge color referenced an undefined token path. This also gives operators and tests a stable way to distinguish connected, partial, disconnected, and empty graph states without parsing SVG geometry.

## Validation
- pnpm --dir dashboard exec vitest run src/components/common/data-flow-graph.test.ts src/components/common/data-flow-graph.a11y.test.ts
- pnpm --dir dashboard exec tsc --noEmit --pretty false
- git diff --check
- pnpm --dir dashboard run lint (passes with 3 existing warnings outside this component)
- pnpm --dir dashboard run build (passes with existing Vite dynamic/static import and chunk-size warnings)
- Browser QA via Vite on 5208: desktop 820x680 and mobile 390x640 screenshots; no horizontal overflow; connected/partial/disconnected/empty metadata checked; default edge computed stroke rgb(212, 161, 74); no page errors

## Review focus
- DataFlowGraph status semantics for missing/non-renderable edges
- SVG stroke fallback behavior for default and custom colored edges
- Metadata attribute names for downstream dashboard QA

### CODE REVIEW

File: dashboard/src/components/common/data-flow-graph.ts [hunk 1]
```diff
@@ -28,14 +28,85 @@ interface DataFlowGraphProps {
...
+    return [{ edge, sourceNode, targetNode, value: normalizedEdgeValue(edge) }]
+  })
+}
+
+export function summarizeDataFlowGraph(nodes: FlowNode[], edges: FlowEdge[]): DataFlowGraphSummary {
+  const renderableEdges = getRenderableFlowEdges(nodes, edges)
+  const validEdgeCount = renderableEdges.length
+  const missingEdgeCount = edges.length - validEdgeCount
+  const totalValue = renderableEdges.reduce((sum, item) => sum + item.value, 0)
+  const maxValue = Math.max(...renderableEdges.map(item => item.value), 1)
```
[the remark this answers is not in the document]
Comment (jeong-sik) [reply dashboard/src/components/common/data-flow-graph.ts]: 동의 — `Math.max(...arr)` spread는 large arrays에서 (a) engine argument limit 충돌 가능, (b) 임시 array allocate. fix:
```ts
const maxValue = renderableEdges.reduce(
  (max, edge) => Math.max(max, edge.value),
  1
);
```
O(n) without spread + no allocation. Engine arg limit 무관.

File: dashboard/src/components/common/data-flow-graph.ts [hunk 2]
```diff
@@ -71,39 +175,51 @@ export function DataFlowGraph({ nodes, edges, onSelectNode, testId }: DataFlowGr
                 key=${`${e.source}-${e.target}`}
                 d="M${sx},${sy} C${cx1},${sy} ${cx2},${ty} ${tx},${ty}"
                 fill="none"
-                stroke=${e.color || 'var(--color-accent)'}
+                stroke=${e.color}
+                style=${{ stroke: e.color || 'var(--accent)' }}
```
[the remark this answers is not in the document]
Comment (jeong-sik) [reply dashboard/src/components/common/data-flow-graph.ts]: 동의 — `stroke` attribute + inline `style.stroke` 둘 다 = inline override. fix: 한 source 만 사용:
```tsx
// CSS variable 표준
<line stroke={`var(--edge-stroke-${kind}, ${defaultStroke})`} />
```
CSS variable로 theming 가능. inline style 제거. 이는 #13499 inline gridTemplateColumns + 본 instance가 같은 axis (CSS specificity).
